src/features/resize-columns/js/ui-grid-column-resizer.js

```diff
@@ -515,21 +515,29 @@
           // Get the cell contents so we measure correctly. For the header cell we have to account for the sort icon and the menu buttons, if present
           var cells = renderContainerElm.querySelectorAll('.' + uiGridConstants.COL_CLASS_PREFIX + col.uid + ' .ui-grid-cell-contents');
           Array.prototype.forEach.call(cells, function (cell) {
-              // Get the cell width
-              // gridUtil.logDebug('width', gridUtil.elementWidth(cell));
 
               // Account for the menu button if it exists
               var menuButton;
               if (angular.element(cell).parent().hasClass('ui-grid-header-cell')) {
                 menuButton = angular.element(cell).parent()[0].querySelectorAll('.ui-grid-column-menu-button');
               }
+              // Make the element float since it's a div and can expand to fill its container
+              // include the cell's font properties since they affect the width
+              var cellElement = angular.element(cell);
 
-              gridUtil.fakeElement(cell, {}, function(newElm) {
-                // Make the element float since it's a div and can expand to fill its container
-                var e = angular.element(newElm);
-                e.attr('style', 'float: left');
+              var cellStyle = window.getComputedStyle(cellElement[0]);
 
-                var width = gridUtil.elementWidth(e);
+              var style = {'float':'left'};
+              for ( var i in cellStyle ) {
+                if(i.startsWith('font')){
```

Comment:
This might be overkill but startsWith is case sensitive so a selector using the property FONT: 'val here' instead of font: 'val here' would not be caught.

Also we might want to explicitly check the return value `i.startsWith('font') !== 0` I am not really sure what our standards are for relying on coercion.

Also if the issue is just that we do not want to override styles we might be able to just use `e.css('float', 'left')`